ki-shell/src/main/kotlin/org/jetbrains/kotlinx/ki/shell/parser/KotlinParserListenerForHighlighting.kt

```diff
@@ -6,6 +6,11 @@ import org.antlr.v4.runtime.tree.ErrorNode
 import org.antlr.v4.runtime.tree.TerminalNode
 import org.jetbrains.kotlinx.ki.shell.parser.KotlinParser.*
 
+private const val IF_EXPR_OFFSET = 2
```

Comment:
Please add the source of the magic constants, i.e. "these offsets are taken from official grammar <url> for navigating to the sub-elements, not directly represented in it.

Reply:
Done